src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -3567,22 +3615,18 @@ void InitIntSolarDistribution(EnergyPlusData &state)
             int const ConstrNum = Surface(SurfNum).Construction;
 
             Real64 AbsIntSurf = state.dataConstruction->Construct(ConstrNum).InsideAbsorpSolar; // Inside opaque surface solar absorptance
-            Real64 AbsIntSurfVis =
-                state.dataConstruction->Construct(ConstrNum).InsideAbsorpSolar; // Inside opaque surface visible absorptance to fix CR 8695 change to
-                                                                                // this = Construct(ConstrNum)%InsideAbsorpVis
-            Real64 HMovInsul = 0.0; // Solar absorptance of outermost layer (or movable insulation if present)
-            Real64 AbsInt;          // Inside opaque surface solar absorptance
-            Real64 AbsExt;          // Solar absorptance of outermost layer (or movable insulation if present)
-            int RoughIndexMovInsul; // Roughness index of movable insulation
-            if (Surface(SurfNum).MaterialMovInsulInt > 0) EvalInsideMovableInsulation(state, SurfNum, HMovInsul, AbsInt);
-            if (HMovInsul > 0.0) AbsIntSurf = AbsInt;
+            Real64 AbsIntSurfVis = state.dataConstruction->Construct(ConstrNum).InsideAbsorpSolar; // Inside opaque surface visible absorptance to fix CR 8695 change to
+
+            if (state.dataSurface->SurfMovInsulHInt(SurfNum) > 0) {
+                AbsIntSurf = state.dataSurface->SurfMovInsulAbsInt(SurfNum);
+            }
```

Comment:
Like explained in the previous comment, `AbsorpSolar` and `AbsorpThermal` are saved to runtime HB arrays. I removed `SurfMovInsulAbsInt` array here and added `SurfAbsSolarInt` instead.

Reply:
In develop branch, `AbsIntSurfVis` is set as `Construct(ConstrNum).InsideAbsorpSolar` instead of `InsideAbsorpVis`, and is not reset when movable insulation is on. Although the comment here says "Inside opaque surface visible absorptance to fix CR 8695 change to this = Construct(ConstrNum)%InsideAbsorpVis". I wonder if this is a bug or it's intended to be fixed like this. @mjwitte Could you help take a look?